cmd/oidc-token-verifier/main.go

```diff
@@ -46,51 +44,37 @@ func NewRootCmd() *cobra.Command {
 	}
 	rootCmd.PersistentFlags().StringVarP(&opts.token, "token", "t", "", "OIDC token")
 	rootCmd.PersistentFlags().StringVarP(&opts.newPublicKeysVarName, "new-keys-var", "n", "OIDC_NEW_PUBLIC_KEYS", "Name of the environment variable to set when new public keys are fetched")
-	rootCmd.PersistentFlags().StringSliceVarP(&opts.trustedWorkflows, "trusted-workflows", "w", []string{}, "List of trusted workflows")
-	err := rootCmd.MarkPersistentFlagRequired("trusted-workflows")
-	if err != nil {
-		panic(err)
-	}
+	// This flag should be enabled once we add support for it in the code.
+	// rootCmd.PersistentFlags().StringSliceVarP(&opts.trustedWorkflows, "trusted-workflows", "w", []string{}, "List of trusted workflows")
+	// err := rootCmd.MarkPersistentFlagRequired("trusted-workflows")
+	// if err != nil {
+	// 	panic(err)
+	// }
```

Comment:
```suggestion
```

When it will be needed it can be taken from Git history.

Reply:
The person who will be working on implementation probably will not remember there is something in git history. This is not a leftover from development, but something we will implement later.